src/Admin.php

```diff
@@ -129,7 +129,7 @@ public static function getProductShortestDeliveryTime(\WC_Product $product): str
    * @implements updated_post_meta
    */
   public static function updateSalePercentage($check, $object_id, $meta_key, $meta_value) {
-    if ($meta_key === '_regular_price' || $meta_key === '_sale_price') {
+    if (in_array($meta_key, ['price', '_regular_price', '_sale_price'])) {
```

Comment:
As we [discussed on Slack](https://netzstrategen.slack.com/archives/CK598LKEV/p1585905605009500), I'm a bit afraid of triggering the ´updateSalePercentage´ method every time a price changes, even if the product is (or was not) in sale.

But, as I understand from @juanlopez4691 clarification, we can do anything different since when a product sale period ends, WooCommerce is changing the `price` meta key, and not the `_regular_price` or `_sale_price` one.